CreateService/index.ts

```diff
@@ -1,77 +1,77 @@
-import { Context } from "@azure/functions";
-
-import * as express from "express";
-import * as winston from "winston";
-
-import { DocumentClient as DocumentDBClient } from "documentdb";
-
-import {
-  SERVICE_COLLECTION_NAME,
-  ServiceModel
-} from "io-functions-commons/dist/src/models/service";
-
-import {
-  TelemetryClient,
-  wrapCustomTelemetryClient
-} from "io-functions-commons/dist/src/utils/application_insights";
-import * as documentDbUtils from "io-functions-commons/dist/src/utils/documentdb";
-import { getRequiredStringEnv } from "io-functions-commons/dist/src/utils/env";
-import { secureExpressApp } from "io-functions-commons/dist/src/utils/express";
-import { AzureContextTransport } from "io-functions-commons/dist/src/utils/logging";
-import { setAppContext } from "io-functions-commons/dist/src/utils/middlewares/context_middleware";
-
-import createAzureFunctionHandler from "io-functions-express/dist/src/createAzureFunctionsHandler";
-
-import { CreateService } from "./handler";
-
-// Whether we're in a production environment
-const isProduction = process.env.NODE_ENV === "production";
-
-const getCustomTelemetryClient = wrapCustomTelemetryClient(
-  isProduction,
-  new TelemetryClient()
-);
-
-const cosmosDbUri = getRequiredStringEnv("CUSTOMCONNSTR_COSMOSDB_URI");
-const cosmosDbKey = getRequiredStringEnv("CUSTOMCONNSTR_COSMOSDB_KEY");
-const cosmosDbName = getRequiredStringEnv("COSMOSDB_NAME");
-
-const documentDbDatabaseUrl = documentDbUtils.getDatabaseUri(cosmosDbName);
-const servicesCollectionUrl = documentDbUtils.getCollectionUri(
-  documentDbDatabaseUrl,
-  SERVICE_COLLECTION_NAME
-);
-
-const documentClient = new DocumentDBClient(cosmosDbUri, {
-  masterKey: cosmosDbKey
-});
-
-const serviceModel = new ServiceModel(documentClient, servicesCollectionUrl);
-
-// tslint:disable-next-line: no-let
-let logger: Context["log"] | undefined;
-const contextTransport = new AzureContextTransport(() => logger, {
-  level: "debug"
-});
-winston.add(contextTransport);
-
-// Setup Express
-const app = express();
-secureExpressApp(app);
-
-// Add express route
-app.post(
-  "/adm/services",
-  CreateService(getCustomTelemetryClient, serviceModel)
-);
-
-const azureFunctionHandler = createAzureFunctionHandler(app);
-
-// Binds the express app to an Azure Function handler
-function httpStart(context: Context): void {
-  logger = context.log;
-  setAppContext(app, context);
-  azureFunctionHandler(context);
-}
-
-export default httpStart;
+import { Context } from "@azure/functions";
```

Comment:
it looks like these files differs only for newlines, may you revert them ?

Reply:
I'm sorry, I tried to revert, but I couldn't manage. 😕

Reply:
can you try removing the file and checkin it out again ? (or copyingit from another clone)

Reply:
I tried to copy it from another clone, also changing the [`core.autocrlf` git setting](https://git-scm.com/book/en/v2/Customizing-Git-Git-Configuration#_formatting_and_whitespace) for the project, but I couldn't manage to revert the line endings anyway... 😕